pkg/expression/infer_pushdown.go

```diff
@@ -221,6 +221,17 @@ func scalarExprSupportedByTiKV(sf *ScalarFunction) bool {
 		/*ast.InetNtoa, ast.InetAton, ast.Inet6Ntoa, ast.Inet6Aton, ast.IsIPv4, ast.IsIPv4Compat, ast.IsIPv4Mapped, ast.IsIPv6,*/
 		ast.UUID:
 
+		return true
+	// Rust use the llvm math functions, which have different precision with Golang/MySQL(cmath)
+	// open the following switchers if we implement them in coprocessor via `cmath`
+	case ast.Conv:
+		arg0 := sf.GetArgs()[0]
+		// To be aligned with MySQL, tidb handles hybrid type argument specially, tikv can't be consistent with tidb now.
+		if f, ok := arg0.(*ScalarFunction); ok {
+			if f.FuncName.L == ast.Cast && f.GetArgs()[0].GetType().Hybrid() {
```

Comment:
Don't need to check `IsBinaryLiteral(f.GetArgs()[0])` here?

Reply:
Not quite sure if constant folding may not take effect. Added for safety.